manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -506,11 +506,14 @@ properties:
       auctioneer:
         api_url: http://auctioneer.service.cf.internal:9016
       etcd:
-        machines: [(( grab properties.etcd.advertise_urls_dns_suffix ))]
-        ca_cert: (( grab meta.secrets.consul_ca_cert ))
-        client_cert: (( grab secrets.consul_agent_cert ))
-        client_key: (( grab secrets.consul_agent_key ))
+        machines: []
+        ca_cert: ""
```

Comment:
Unfortunately diego-release [expects it](https://github.com/cloudfoundry/diego-release/blob/v0.1487.0/jobs/bbs/templates/bbs_ca.crt.erb) and there is [no default](https://github.com/cloudfoundry/diego-release/blob/v0.1487.0/jobs/bbs/spec#L68-L69)